.travis.yml

```diff
@@ -28,6 +28,7 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - pip install https://github.com/mattgwwalker/msg-extractor/tarball/master
```

Comment:
Why not add this to requirements.txt?

Reply:
because it's optional and you only need it if you joined the dark side: https://github.com/OCA/social/pull/242/files#diff-7030e17fbd71188feaab159c70cc100cR11

Reply:
hahaha, OK. I'm from the Jedi side also.